<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>1920</width>
     <height>1080</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="Set">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="Reset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="FlipPositionLine">
        <property name="inputMask">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLineEdit" name="To_UllongLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="FlipPosition">
        <property name="text">
         <string>Flip(position(below))</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QPushButton" name="GetSize">
        <property name="text">
         <string>GetSize</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="SetPosition">
        <property name="text">
         <string>Set(position(below))</string>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QLineEdit" name="GetSizeLine">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="Any">
        <property name="text">
         <string>Any</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="SetBitset">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Set BitSet</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="Flip">
        <property name="text">
         <string>Flip</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="5">
       <widget class="QLineEdit" name="BitSet">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="SetBitsetLine">
        <property name="inputMask">
         <string comment="8">00000000</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLineEdit" name="SetPositionLine">
        <property name="inputMask">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QLineEdit" name="ResetPositionLine">
        <property name="inputMask">
         <string>00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="All">
        <property name="text">
         <string>All</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="ResetPosition">
        <property name="text">
         <string>Reset(position(below))</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="To_Ullong">
        <property name="text">
         <string>To ullong</string>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="None">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>913</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
